src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+export const PeoplePage: FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(false);
  const [isError, setIsError] = useState<boolean>(false);
```
Why did you set the type for isLoading and not for isError?
Remove or add it in both cases